packages/@aws-cdk/aws-cloudwatch/test/test.metrics.ts

```diff
@@ -231,4 +231,14 @@ export = {
 
     test.done();
   },
+
+  'metric accepts a variety of statistics'(test: Test) {
+    new Metric({
+      namespace: 'Test',
+      metricName: 'Metric',
+      statistic: 'myCustomStatistic',
+    });
+
+    test.done();
+  },
```

Comment:
This test feels insufficient. You should verify that 'myCustomStatistic' is actually used by plugging into an alarm or widget.